tests/utils/config_babe.toml

```diff
@@ -0,0 +1,37 @@
+[global]
+name = "gssmr"
+id = "gssmr"
+basepath = "~/.gossamer/gssmr"
+log = "crit"
+
+[log]
+core = "info"
+babe = "info"
+
+[init]
+genesis = "./chain/gssmr/genesis.json"
+
+[account]
+key = ""
+unlock = ""
+
+[core]
+authority = true
+roles = 4
+babe-authority = true
+grandpa-authority = true
+
+[network]
+port = 7001
+bootnodes = []
+protocol = "/gossamer/gssmr/0"
+nobootstrap = false
+nomdns = false
+
+[rpc]
+enabled = false
+port = 8545
+host = "localhost"
+modules = ["system", "author", "chain", "state"]
+ws-port = 8546
+ws-enabled = false
```

Comment:
These config files look great for defining default configurations.  Do we need to include RPC parameters here since it seems that they are overwritten in StartGossamer function?

Reply:
yeah I can remove them, I just copied these from the default and changed only what I needed lol